_data/other.yml

```
@@ -353,15 +353,15 @@ websites:
     url: https://nostarch.com/
     img: nostarchpress.png
     tfa:
-      - software
+      - proprietary
```

https://github.com/2factorauth/twofactorauth/pull/3246
Looks like they support totp
```suggestion
      - totp
```